packages/firestore/test/integration/util/helpers.ts

```diff
@@ -23,6 +23,9 @@ import firebase from './firebase_export';
 import { EmptyCredentialsProvider } from '../../../src/api/credentials';
 import { PlatformSupport } from '../../../src/platform/platform';
 import { AsyncQueue } from '../../../src/util/async_queue';
+import { Firestore, FirestoreDatabase } from '../../../src/api/database';
+import { FirebaseApp } from '../../../../app/index';
```

Comment:
This will break, if you are trying to reference other modules you need to do it directly

_i.e._
```javascript
import { FirebaseApp } from '@firebase/app';
```

Reply:
Done. Thanks for debugging this!